src/Scene.js

```diff
@@ -268,6 +281,20 @@ class Scene extends React.Component<SceneProps, {}> {
             </React.Fragment>
         );
     }
+
+    componentDidUpdate(prevProps: SceneProps, prevState: SceneState) {
+        if (prevProps.runningState !== this.props.runningState) {
```

Comment:
This is not sufficient to ensure that position changes are announced before the program has been run.  You'll also need to check to see if the character position/orientation within the scene have changed.  As it stands, changes that result from using the position controls are not announced until you run the program at least once.